rasa/core/brokers/pika.py

```diff
@@ -168,7 +169,9 @@ async def _connect(self) -> aio_pika.RobustConnection:
         # The `url` parameter will take precedence over parameters like `login` or
         # `password`.
         if self.host.startswith("amqp"):
-            url = self.host
+
+            host_ob = urlparse(self.host)
```

Comment:
What does the name `host_ob` mean in this context? Can there be a better name for this?

Reply:
@alwx 
I almost lost hope of you reviewing this PR after 6 months!!!!
Thanks for the update.
I renamed `host_ob` to `parsed_host`, does that make sense for you?  
This object is just for holding the parsed url components values.

Reply:
Sounds better now, thanks!